packages/material-ui/src/useScrollTrigger/useScrollTrigger.test.js

```diff
@@ -68,7 +68,7 @@ describe('useScrollTrigger', () => {
       assert.strictEqual(values.callCount, 1);
     });
 
-    it('should be false by default when using setRef', () => {
```

Comment:
I don't understand how this relates to `setRef`. Probably resolved during PR review?